recipes/colorpedia/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "colorpedia" %}
+{% set version = "1.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/colorpedia-{{ version }}.tar.gz
+  sha256: 13e2536c8afe8e9d98b5966c00558be6778ff776952057c97b452be20c0fa3fa
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - color = colorpedia.cli:entry_point_color
+    - colorpedia = colorpedia.cli:entry_point_colorpedia
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - fire >=0.3.1
+
+test:
+  imports:
+    - colorpedia
+  commands:
+    - pip check
```

Comment:
Pip is not working well here, we can remove it
```suggestion
```

Reply:
Ok got it. Thank you.